src/main/java/ch/wisv/areafiftylan/users/model/User.java

```diff
@@ -75,6 +77,19 @@
     @JsonIgnore
     private boolean enabled = true;
 
+    public User(String email, String passwordHash) {
+        this.email = email;
+        this.passwordHash = passwordHash;
+        this.profile = new Profile();
+        this.roles = new HashSet<>();
```

Comment:
Previously you initialized it with `Sets.newHashSet(Role.ROLE_USER)`, intended to be removed? Also you already initialized this field, so I don't think again initializing is required